qa/tasks/mgr/dashboard/test_osd.py

```diff
@@ -220,7 +221,7 @@ def get_destroy_status():
     def test_osd_devices(self):
         data = self._get('/api/osd/0/devices')
         self.assertStatus(200)
-        self.assertSchema(data, JList(JObj({
+        self.assertSchema(data, JList(JObj({  # pylint: disable=R0801
```

Comment:
with `addopts = -rn --rcfile=.pylintrc --jobs={[pylint]jobs} --disable=duplicate-code` I think this is no longer necessary:
```suggestion
        self.assertSchema(data, JList(JObj({
```
Same for other occurences of `# pylint: disable=R0801`

Reply:
Oh, yes, you're right. Thank you!